### DISCUSSION TITLE
checks in  set_class method : keep or remove

### DISCUSSION THREAD

**Discussion by sandeep-gh** [elimintz/justpy#562] [Polls]
There are checks in set_class method that uses predefined sets of tailwind constructs. It seems a bit outdated (rose, sky etc. colors not present, outline classes not there, outline-shadow is there but its no longer part of tailwind). 
My suggestion would be to let go of this checks, since its bit cumbersome to manage.

**Comment by elimintz** [comment]
I agree. This is becoming too cumbersome and should be left to the individual developer just as is done in JavaScript frameworks
